sdk/elixir/lib/dagger/gen/container.ex

```diff
@@ -729,6 +729,20 @@ defmodule Dagger.Container do
           arg(selection, "owner", optional_args[:owner])
         end
 
+      selection =
+        if is_nil(optional_args[:optional]) do
+          selection
+        else
+          arg(selection, "Optional", optional_args[:optional])
```

Comment:
I think it is because of https://github.com/dagger/dagger/pull/5707/files#r1309097818 and https://github.com/dagger/dagger/pull/5707/files#r1309097978